webapp/src/main/webapp/src/app/shared/base.service.ts

```diff
@@ -25,4 +25,12 @@ export abstract class BaseService<T extends BaseEntity> {
 		return this.httpClient.delete<T>(environment.apiBaseUrl + resource + '/' + entity.id);
 	}
 
+	postCountAndItems(resource: string, data: any): Observable<BaseCountAndItems<T>> {
```

Comment:
Why do you have this method? Where is it used? And why is it a post?

Reply:
I discard that.